com.unity.netcode.gameobjects/Prototyping/Interpolator/BufferedLinearInterpolator.cs

```diff
@@ -0,0 +1,232 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Unity.Netcode
+{
+    /// <summary>
+    /// Solves for incoming values that are jittered
+    /// Partially solves for message loss. Unclamped lerping will help hide it, but not completely
```

Comment:
Code comment out of date partially since this uses unclamped lerp?

Reply:
```suggestion
    /// Partially solves for message loss. Unclamped lerping helps hide this, but not completely
```
not out of date, badly formulated :P Will update